website/static/files/ads_orca_schema_refined.xed

```diff
@@ -162,46 +163,43 @@
 <Storage name="WithCRCols" value="false"/>
 <Storage name="External" value="false"/>
 </Table>
-<Table id="35" name="files" schema="orca" database="disaster_recovery" background="16777165">
+<Table id="36" name="files" schema="orca" database="disaster_recovery" background="16777165">
 <Text>Files that are in the ORCA holdings. (Latest version only)</Text>
-<Column id="36" name="id" type="int8" length="25" nulls="false">
+<Column id="37" name="id" type="int8" length="25" nulls="false">
 <CP name="COMMENTS">Internal ORCA file ID</CP>
 </Column>
-<Column id="37" name="granule_id" type="int8" length="25" nulls="false">
+<Column id="38" name="granule_id" type="int8" length="25" nulls="false">
 <CP name="COMMENTS">Granule that the file belongs to refrences the internal ORCA granule ID.</CP>
 </Column>
-<Column id="38" name="name" type="text" length="25" nulls="false">
+<Column id="39" name="name" type="text" length="25" nulls="false">
 <CP name="COMMENTS">Name of the file including extension</CP>
 </Column>
-<Column id="39" name="orca_archive_location" type="text" length="25" nulls="false">
+<Column id="40" name="orca_archive_location" type="text" length="25" nulls="false">
 <CP name="COMMENTS">S3 Glacier bucket that the file object is stored in</CP>
 </Column>
-<Column id="40" name="cumulus_archive_location" type="text" length="25" nulls="false">
+<Column id="41" name="cumulus_archive_location" type="text" length="25" nulls="false">
 <CP name="COMMENTS">Cumulus S3 bucket where the file is thought to be stored.</CP>
 </Column>
-<Column id="41" name="key_path" type="text" length="25" nulls="false">
+<Column id="42" name="key_path" type="text" length="25" nulls="false">
 <CP name="COMMENTS">Full AWS key path including file name of the file (does not include bucket name) where the file resides in ORCA.</CP>
 </Column>
-<Column id="42" name="multipart_chunksize_mb" type="integer" length="25" nulls="true">

```

Comment:
Is this removal intentional?

Reply:
yes, since I added a new column `cumulus_createdAt`, so this one got move to 
`<Column id="43" name="multipart_chunksize_mb" type="integer" length="25" nulls="true"/>`

Reply:
And we're fine with losing the "PRECISION" and "SCALE" properties?

Reply:
this is the file Aqua data studio created for me so I assume that are not needed since I can open this file without any issue.

Reply:
Intger would not have a precision or scale.